pkg/clusteragent/autoscaling/workload/controller_horizontal.go

```diff
@@ -133,13 +134,15 @@ func (hr *horizontalController) performScaling(ctx context.Context, podAutoscale
 	if err != nil {
 		err = fmt.Errorf("failed to scale target: %s/%s to %d replicas, err: %w", scale.Namespace, scale.Name, horizontalAction.ToReplicas, err)
 		hr.eventRecorder.Event(podAutoscaler, corev1.EventTypeWarning, model.FailedScaleEventReason, err.Error())
+		telemetryHorizonalScale.Inc(scale.Namespace, scale.Name, strconv.Itoa(int(horizontalAction.ToReplicas)), strconv.Itoa(int(horizontalAction.FromReplicas)), strconv.Itoa(int(*horizontalAction.RecommendedReplicas)), "true")
 		autoscalerInternal.UpdateFromHorizontalAction(nil, err)
 		return autoscaling.Requeue, err
 	}
 
 	log.Debugf("Scaled target: %s/%s from %d replicas to %d replicas", scale.Namespace, scale.Name, horizontalAction.FromReplicas, horizontalAction.ToReplicas)
 	autoscalerInternal.UpdateFromHorizontalAction(horizontalAction, nil)
 	hr.eventRecorder.Eventf(podAutoscaler, corev1.EventTypeNormal, model.SuccessfulScaleEventReason, "Scaled target: %s/%s from %d replicas to %d replicas", scale.Namespace, scale.Name, horizontalAction.FromReplicas, horizontalAction.ToReplicas)
+	telemetryHorizonalScale.Inc(scale.Namespace, scale.Name, strconv.Itoa(int(horizontalAction.ToReplicas)), strconv.Itoa(int(horizontalAction.FromReplicas)), strconv.Itoa(int(*horizontalAction.RecommendedReplicas)), "false")
```

Comment:
Currently you're tracking _applied_ recommendations, but not received recommendations. We're interested in both